Makefile

```diff
@@ -5,26 +5,24 @@ IMG ?= kudobuilder/controller:${TAG}
 EXECUTABLE := manager
 CLI := kubectl-kudo
 
-all: test manager
+export GO111MODULE=on
 
-deps:
-	go install github.com/kudobuilder/kudo/vendor/github.com/golang/dep/cmd/dep
-	dep check -skip-vendor
-	go install github.com/kudobuilder/kudo/vendor/golang.org/x/tools/cmd/goimports
-	go install github.com/kudobuilder/kudo/vendor/golang.org/x/lint/golint
-	go get -u honnef.co/go/tools/cmd/staticcheck
+all: test manager
 
 # Run tests
-test: generate deps fmt vet lint imports manifests
-	go test ./pkg/... ./cmd/... -coverprofile cover.out
+test:
```

Comment:
absolutely need `.PHONY: test` . 
make can't `make test` now that there is a `test` directory.   Probably need `.PHONY:` with all of our targets.  It is recommended to add a line `.PHONY <x>` before each target.

Reply:
and thank you for having all the test pass!!! :)